src/pages/public_registration/ChapterPublisherInfo.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { t } from 'i18next';
+import { fetchResults } from '../../api/searchApi';
+import { BookPublicationContext } from '../../types/publication_types/bookRegistration.types';
+import { ChapterPublicationContext } from '../../types/publication_types/chapterRegistration.types';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { PublicPublisher } from './PublicPublicationContext';
+
+interface ChapterPublisherInfoProps {
+  publicationContext: ChapterPublicationContext;
+}
+
+export const ChapterPublisherInfo = ({ publicationContext }: ChapterPublisherInfoProps) => {
+  const publisherIdentifier = publicationContext.id && getIdentifierFromId(publicationContext.id);
+
+  const publisherQuery = useQuery({
+    enabled: !!publisherIdentifier,
+    queryKey: ['publisher', publisherIdentifier],
+    queryFn: () => fetchResults({ id: publisherIdentifier, results: 1 }),
+    meta: { errorMessage: t('feedback.error.search') },
+  });
+
+  const publisherQueryResult = publisherQuery.data?.hits[0];
+
+  const publisher =
+    publisherQueryResult &&
+    (publisherQueryResult?.entityDescription?.reference?.publicationContext as BookPublicationContext).publisher;
```

Comment:
Lurer på om `BookPublicationContext` kan føre til feil også.. 🤔 
Har du prøvd med `Kapittel i rapport` feks? Da vil parent/publisher være `Report`, og det vil følgelig være `ReportPublicationContext`
```suggestion
    (publisherQueryResult?.entityDescription?.reference?.publicationContext as BookPublicationContext | ReportPublicationContext).publisher;
```

Reply:
Om du sjekker [ChapterForm.tsx](https://github.com/BIBSYSDEV/NVA-Frontend/blob/f5eb4b084551ca8f83ac73185c7a8f3644bf8711/src/pages/registration/resource_type_tab/sub_type_forms/ChapterForm.tsx#L44) så ser du at publisher kan være `BookAnthology`, eller hvilken som helst `Report`-type